var/spack/repos/builtin/packages/py-libensemble/package.py

```diff
@@ -49,7 +50,7 @@ class PyLibensemble(PythonPackage):
 
     depends_on("py-numpy", type=("build", "run"))
     depends_on("py-psutil", type=("build", "run"), when="@0.7.1:")
-    depends_on("py-setuptools", type=("build", "run"))
+    depends_on("py-setuptools", type=("build"))
```

Comment:
It's still needed in older versions:
```suggestion
    depends_on("py-setuptools", when="@0.10.2:", type="build")
    depends_on("py-setuptools", when="@:0.10.1", type=("build", "run"))
```